src/assets/js/script.js

```diff
@@ -0,0 +1,37 @@
+const modal = document.querySelector(".modal");
+const modalImg = document.querySelector(".modal__content");
+
+document.querySelector(".modal").addEventListener("click", () => {
+  modal.classList.remove("showModal");
+});
+
+document
+  .querySelector(".work__content")
+  .addEventListener("click", function (event) {
+    let targetImg = event.target;
+
+    if (targetImg.tagName != "IMG") return;
```

Comment:
Please start every code from new line and wrap it in curly braces.

Reply:
``` suggestion 
    if (targetImg.tagName !== "IMG") {
      return;
    } else {
      modal.classList.add("showModal");
      modalImg.setAttribute("src", targetImg.src);
    }
```